package.json

```diff
@@ -23,6 +23,7 @@
     "@tzkt/sdk-api": "^2.2.0",
     "eslint-config-next": "^12.0.4",
     "framer-motion": "^10.12.16",
+    "lint": "^0.8.19",
```

Comment:
What's the purpose of lint here? If it's used for linting then it should be under "devDependency".